cellprofiler/modules/identifysecondaryobjects.py

```diff
@@ -13,6 +13,15 @@
 import cellprofiler.module
 import cellprofiler.object
 import cellprofiler.setting
+
+import numpy as np
+
+import cellprofiler.image as cpi
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.preferences as cpprefs
+import cellprofiler.setting as cps
+from cellprofiler.setting import YES, NO
```

Comment:
These import aliases have been removed, and the full imports are declared above. You can delete lines 16-24.